pype/plugins/tvpaint/publish/validate_marks.py

```diff
@@ -0,0 +1,66 @@
+import json
+
+import pyblish.api
+from avalon.tvpaint import lib
+
+
+class ValidateMarksRepair(pyblish.api.Action):
+    """Repair the marks."""
+
+    label = "Repair"
+    icon = "wrench"
+    on = "failed"
+
+    def process(self, context, plugin):
+        expected_data = ValidateMarks().get_expected_data(context)
```

Comment:
Rather not create new object of `ValidateMarks` but convert `get_expected_data` to `classmethod` or `staticmethod`. Initialization may change over time and it's not save to create an object just to call a function which does not need a created object.

Reply:
Good point. Done